### PR TITLE
feat: gitea otomi charts repo

### PR DESCRIPTION
**Pull request by ferruhcihan**
### Implements:
https://app.zenhub.com/workspaces/dev-5e7e84dddc966f05a627a19b/issues/gh/redkubes/unassigned-issues/654

### Description:
This PR adds a feature to clone github otomi-charts repository and push to gitea. User can use the gitea charts repository for auto image updates.
Is paired with: https://github.com/redkubes/otomi-tasks/pull/88

### Changes Made:
Added cloneOtomiChartsInGitea function to call at the end of the apply command.

## Checklist

- [ ] Architecture Design Records have been added as `adr/*.md` and appended to list in `adr/_index.md`, if applicable.
- [ ] The `values-schema.yaml` file and `test/**` fixtures have been updated to reflect code changes, if applicable.
- [ ] The OpenApi Schema from redkubes/otomi-api project is compatible with definitions from `values-schema.yaml` file, if applicable.
- [ ] Helm releases are meeting otomi's baseline security policies, if applicable.
- [ ] Helm chart and helmfile changes are tested against upgrade scenario, if applicable.

### CODE REVIEW

File: src/cmd/commit.ts [hunk 1]
```diff
@@ -63,6 +63,33 @@ export const commit = async (): Promise<void> => {
   await commitAndPush(values, branch)
 }
 
+export const cloneOtomiChartsInGitea = async (): Promise<void> => {
+  const d = terminal(`cmd:${cmdName}:gitea-otomi-charts`)
+  d.info('Cloning otomi-charts in Gitea')
+  const values = (await hfValues()) as Record<string, any>
+  const { email, username, password } = getRepo(values)
+  const workDir = '/tmp/otomi-charts'
+  const otomiChartsUrl = 'https://github.com/redkubes/otomi-charts.git'
```
Comment (j-zimnowoda) [inline src/cmd/commit.ts:72]: Let's make it configurable through env with the default to `https://github.com/redkubes/otomi-charts.git`

**Review by ElderMatt — COMMENTED** [review]
When Jeho's comment is resolved it looks good

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]
There is just one comment to be covered

File: src/cmd/commit.ts [hunk 2]
```diff
@@ -69,7 +69,7 @@ export const cloneOtomiChartsInGitea = async (): Promise<void> => {
   const values = (await hfValues()) as Record<string, any>
   const { email, username, password } = getRepo(values)
   const workDir = '/tmp/otomi-charts'
-  const otomiChartsUrl = 'https://github.com/redkubes/otomi-charts.git'
+  const otomiChartsUrl = process.env.OTOMI_CHARTS_URL || 'https://github.com/redkubes/otomi-charts.git'
```
Comment (j-zimnowoda) [inline src/cmd/commit.ts:72]: Ferruh, take a look at`src/validators.ts` file.
The env type and its default should be defined there.

File: .env.sample [hunk 3]
```diff
@@ -22,3 +22,5 @@ AWS_ACCESS_KEY_ID=''
 AWS_SECRET_ACCESS_KEY=''
 # Vault:
 VAULT_TOKEN=''
+
+OTOMI_CHARTS_URL=''
```
Comment (j-zimnowoda) [inline .env.sample:26]: For dev purpose let's make it to `https://github.com/redkubes/otomi-charts`.
Then workload creation will be able to go through the all steps

**Review by j-zimnowoda — APPROVED** [review]

**Pull request merged**